middlewares/jwtValidator.js

```diff
@@ -0,0 +1,24 @@
+const jwtUtil = require('../lib/utils/jwtUtil');
+
+const jwtValidator = (req, res, next) => {
+  // Get the token sent from anywhere(i.e header,body or query)
+  const token = req.body.token || req.query.token || req.headers.token || req.params;
+  if (token) {
+    // Decode token if it is not empty and call next function
+    try {
+      req.decodedToken = jwtUtil.verifyToken(token);
+      next();
+    } catch (err) {
+      // Return an error if it occurs
+      res.json({
+        success: false,
+        message: err.message
```

Comment:
Explicitly stating the error that occurred here is a better way to go as it gives us the ability to clearly communicate with the user.

Reply:
The err is coming from jtw library and it has a message property that describes what error that has occured. That's why i did that

Reply:
@tosinibrahim96 Add a `return` before `next()` and also before the `res.status` after catch.

Reply:
done @eltNEG thanks